themes/theme-b2c-tailwind/src/molecules/icon-button.css

```diff
@@ -0,0 +1,3 @@
+[data-custom-store-icon-button] {
+  @apply text-black inline-flex bg-transparent cursor-pointer rounded-full p-2 transition-colors duration-150 ease-in hover:bg-gray-400 hover:bg-opacity-40;
```

Comment:
![Screen Shot 2021-08-18 at 16 16 24](https://user-images.githubusercontent.com/10627086/129958571-6085431d-f772-404b-8091-3d86e6ec5589.png)

the icon is not aligned on the center, maybe if you use `items-center` fix that.

Reply:
Yeah, it's the default one, i created another example stylized.

Reply:
Do you think I also should fix this example?

Reply:
yep.

Reply:
Fixed

Reply:
@igorbrasileiro What do you think about creating another CSS block just for the hover effect? Like this:

```
[data-custom-store-icon-button] {
  @apply text-black inline-flex bg-transparent cursor-pointer rounded-full p-2 transition-colors duration-150 ease-in;
}

[data-custom-store-icon-button]:hover {
    @apply bg-gray-400 bg-opacity-40;
}
```

IMO it has a better readability.

Reply:
Ok @Gmantiqueira.

Reply:
Fixed